core/ecschema-metadata/src/Metadata/OverrideFormat.ts

```diff
@@ -100,4 +101,11 @@ export class OverrideFormat {
         fullName += `[${unit.fullName}|${unitLabel}]`;
     return fullName;
   }
+
+  public static isOverrideFormat(object: any): object is OverrideFormat {

```

Comment:
I don't think we need this one. This class does not derive from the `SchemaItem` so can't be any other type.

Reply:
I mainly added `isOverrideFormat` method because of type checking in `getNameFromArgument()` [link to changes](https://github.com/imodeljs/imodeljs/pull/921/files#diff-9163d1fb55c1ba2246af62a064175580af5d09b859572bdf7c20607db4894358). And precisely because it does not derive from the `SchemaItem` I couldn't think of a cleaner way to check for OverrideFormat type.